airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -794,6 +827,68 @@ private Map<UUID, AirbyteCatalog> findCatalogByHash(final String catalogHash, fi
     return result;
   }
 
+  public List<SourceConnection> getSourceConnections(final List<UUID> sourceIds) throws IOException {
```

Comment:
curious - some of this formatting looks slight shorter than 150 to me. Can you double check your IntelliJ has the Airbyte format set up?

Reply:
Great catch, I think my default got reset to 120 when I re-installed IntelliJ a little while ago due to some gradle module import issues, I set it back to 150, will try to go through and correct some of this formatting